src/views/List.jsx

```diff
@@ -18,29 +24,53 @@ export function List({ data }) {
 		setSearchTerm('');
 	};
 
+	const copyToken = () => {
+		navigator.clipboard.writeText(listToken);
+		setCopied(true);
+	};
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
-			<form onSubmit={(e) => e.preventDefault()}>
-				<label>
-					Filter Items
-					<input
-						type="text"
-						placeholder="start typing here..."
-						id="filter"
-						name="filter"
-						value={searchTerm}
-						onChange={(e) => {
-							setSearchTerm(e.target.value);
-						}}
-					/>
-				</label>
-			</form>
-			<button type="button" onClick={clearSearchTerm}>
-				clear
-			</button>
+			{data.length ? (
+				<div>
+					<label>
+						Filter Items
+						<input
+							type="text"
+							placeholder="start typing here..."
+							id="filter"
+							name="filter"
+							value={searchTerm}
+							onChange={(e) => {
+								setSearchTerm(e.target.value);
+							}}
+						/>
+					</label>
+					<button type="button" onClick={clearSearchTerm} aria-live="polite">
+						clear
+					</button>
+					<div>
+						<p>
+							Copy token to allow others join your list:
+							<button onClick={copyToken} id="token">
+								{copied ? 'Copied!' : listToken}
+							</button>
+						</p>
+					</div>
```

Comment:
Putting the `button` **inside** of the `p` element allows the button and paragraph to be side-by-side visually for now, but in the future I would consider moving the `button` to **outside** of the `p` element and using flexbox to position the paragraph and button